pkgs/tools/networking/mailsend-go/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, fetchFromGitHub, gitUpdater, buildGoModule }:
+let
+  pname = "mailsend-go";
+
+  version = "1.0.10";
+
+  owner = "muquit";
+
+  rev-prefix = "v";
+
+  url = "https://github.com/${owner}/${pname}";
+in
+
+buildGoModule {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = pname;
+    rev = "${rev-prefix}${version}";
```

```suggestion
buildGoModule rec {
  pname = "mailsend-go";
  version = "1.0.10";

  src = fetchFromGitHub {
    inherit "muquit";
    repo = "mailsend-go";
    rev = "v${version}";
```

Generally we do not structure packages like this and prefer simpler and less complex things if possible. For this simple package, we should do it like the suggestion